### PR TITLE
bump zodios to `v10.9.2` and update incorrect API schemas

### PR DESCRIPTION
**Pull request by radeksimko**
This is to reflect the patch upstream https://github.com/ecyrbe/zodios/pull/471

I ran the schemas through all the data we have available in our shared org and they all comply.

### CODE REVIEW

File: src/terraformCloud/run.ts [hunk 1]
```diff
@@ -121,37 +132,46 @@ export const CONFIGURATION_SOURCE: { [id: string]: string } = {
...
 
+const configurationVersionRelationships = z.object({
+  'ingress-attributes': relationship,
+});
+
 // include=configuration_version (implied from .ingress_attributes too)
 // See https://developer.hashicorp.com/terraform/cloud-docs/api-docs/configuration-versions#show-a-configuration-version
 const configurationVersionAttributes = z.object({
-  source: z.enum([cfgSources[0], ...cfgSources]),
+  source: z.enum([cfgSources[0], ...cfgSources]).nullish(),
```
Comment (dbanck) [inline src/terraformCloud/run.ts:146]: `[cfgSources[0], ...cfgSources]` We duplicate the first entry to end up with an array that contains `ado` twice? What's the intention here?

```suggestion
  source: z.enum(cfgSources).nullish(),
```

**Review by dbanck — APPROVED** [review]
LGTM! Only one small question

File: src/terraformCloud/run.ts (hunk 1 above)
Comment (radeksimko) [reply src/terraformCloud/run.ts:146]: That decoupling of the first element was intentional but the inclusion of that same element in the 2nd argument was not. This is purely to work around this oddity

Screenshot 2023-06-13 at 16 04 52

Comment (radeksimko) [reply src/terraformCloud/run.ts:146]: I will fix the exclusion bug.

**Pull request merged**